src/Core/Settings.h

```diff
@@ -584,6 +584,8 @@ static constexpr UInt64 operator""_GiB(unsigned long long value)
     \
     M(Bool, allow_unrestricted_reads_from_keeper, false, "Allow unrestricted (without condition on path) reads from system.zookeeper table, can be handy, but is not safe for zookeeper", 0) \
     \
+    M(Bool, allow_writes_to_zookeeper, false, "Allow writes to system.zookeeper table, can be handy, but is not safe for zookeeper", 0) \
```

Comment:
This does not protect from anything. Please, read previous comments in the conversation carefully.

Reply:
you mean a config item like "allow_zookeeper_write : true/false" in the config file? Well, I was too hurry

Reply:
Then how do we enable it in the tests?

Reply:
I found `01594_too_low_memory_limits.sh` also uses ad-hoc config file, I will look into it

Reply:
`01594_too_low_memory_limits` is bad test that was added by mistake, it should have been integration test. The correct way to enable it in tests is to add xml file to `tests/config/config.d/` and update `tests/config/install.sh`